docs/networks/node-ops/node-operation/node-bootstrap.md

```diff
@@ -178,6 +178,8 @@ $cat ./bootstrap/public-root-information/node-info.pub.39fa54984b8eaa463e1299194
 If you are running a collection or consensus node, you will need to provide an additional field `Machine Account Public Key`.
 This value is found in the output of the `bootstrap key` command from Step 1.
 
+Staking a collection or consensus node will also create a machine account for the node. The machine account will be mentioned in the output of the staking transaction displayed by Flow Port. Please save the machine account for the next step.
```

Comment:
```suggestion
```
I feel like this is saying the same thing as the previous paragraph?

Reply:
The previous paragraph instructs the user to provide the machine account public key when staking however it doesn't mention that the staking transaction will generate a `machine account` which needs to be provided in the next step where it says `YOUR_MACHINE_ACCOUNT_ADDRESS_GOES_HERE`.

Reply:
Right, account address vs key. Makes sense 👍